src/Symfony/Bridge/Doctrine/Form/ChoiceList/IdReader.php

```diff
@@ -138,7 +138,7 @@ public function getIdValue($object)
      *
      * @return string The name of the ID field
      */
-    public function getIdField()
+    public function getIdField(): string
```

Comment:
if `$classMetadata->getIdentifierFieldNames` (called in the constructor) returns an empty array, then $idField will be equals to `false`

I don't know if we should we handle such case?